packages/ember-runtime/lib/mixins/enumerable.js

```diff
@@ -741,7 +741,7 @@ const Enumerable = Mixin.create({
       let method = x && x[methodName];
 
       if ('function' === typeof method) {
-        ret[idx] = args ? method.apply(x, args) : x[methodName]();
+        ret[idx] = args.length ? method.apply(x, args) : x[methodName]();
```

Comment:
@krisselden maybe it should be just `method.apply(x, args)` without args check ?